src/components/AddBook.js

```diff
@@ -1,8 +1,34 @@
-const AddBook = () => (
-  <div className="add-book">
-    <input type="text" placeholder="Add new book" />
-    <button type="submit" value="add-book">Add</button>
-  </div>
-);
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
```

Comment:
Hello @marilirulita.  I've read the conversation you had with the last reviewer. Before adding this comment, I did my research. The answer is:

- [ ] Yes, both the title and the author need their own inputs that update the local state every time they update (onChange).  Kindly implement this so we can meet this particular requirement.

Sorry for the confusion.  The explanation is that the sneak peek image is there for the final milestone while adding styles.

Judging by what you've achieved so far in this project, I'm pretty sure you can do it.
I'm looking forward to your resubmission 💪

Reply:
@Meltrust thanks for answering my question.
Good day!